### PR TITLE
Imported/Fixed Swagger generated code

### PR DESCRIPTION
**Pull request by arawinters**
I had to clean up their code in places but it does seem to work. Also plugged it in to the new ng-packgr style structure so we can easily generate npm modules from it.

### CODE REVIEW

**Comment by arawinters** [comment]
*bump*
anything?

this code is already available via [npm package](https://www.npmjs.com/package/@senzing/rest-api-client-ng)

File: docs/classes/CustomHttpUrlEncodingCodec.html [hunk 1]
```diff
@@ -0,0 +1,348 @@
...
+        <div class="container-fluid main">
+           <div class="row main">
+               <div class="hidden-xs menu">
+                   <compodoc-menu mode="normal"></compodoc-menu>
+               </div>
+               <!-- START CONTENT -->
+               <div class="content class">
+                   <div class="content-data">
+
+
```
Comment (docktermj) [inline docs/classes/CustomHttpUrlEncodingCodec.html]: Are these blank lines from auto-generated code?

Comment (arawinters) [reply docs/classes/CustomHttpUrlEncodingCodec.html]: yup. compodoc spits them out from source code.

**Comment by arawinters** [comment]
> I'm not sure I can contribute much with a review through github. There's too much to go through. Are there any parts that you're specifically concerned about, and how can I view this in swagger's normal display format?

I grabbed a copy of the yaml file and committed it to the project just so we'd know which one was used to gen the code. it'd be:
http://editor.swagger.io/?url=https://raw.githubusercontent.com/Senzing/rest-api-client-ng/pre-alpha/senzing-api.yaml

but no, I'm not concerned about this module. it's mostly generated from the api repo yaml with a few changes just so angular could compile it. I thought about maybe pulling in the swagger gen code and adding a script to build the code right from this repo(as opposed to going to swagger and copying the files in to the repo) but there was quite a bit involved. If we need that functionality we can add it, didn't want to burn through time for something that's functionally equiv.

**Comment by docktermj** [comment]
Issue number: #4

**Pull request merged**